src/com/github/igotyou/FactoryMod/FactoryModManager.java

```diff
@@ -29,13 +29,17 @@
 	private HashSet<Material> possibleInteractionBlock;
 	private Material factoryInteractionMaterial;
 	private boolean citadelEnabled;
+	private int redstonePowerOn;
+	private int redstoneRecipeChange;
 	private String compactLore;
 
 	public FactoryModManager(FactoryModPlugin plugin,
-			Material factoryInteractionMaterial, boolean citadelEnabled) {
+			Material factoryInteractionMaterial, boolean citadelEnabled, int redstonePowerOn, int redstoneRecipeChange) {
```

Comment:
I was thinking other factories types could definitely reuse them, as almost all factories will have (1) on / off and (2) behavior control, which is what these redstone levels control. 

I could be wrong, in which case it's nbd to change.